impeller/typographer/rectangle_packer.cc

```diff
@@ -129,14 +135,14 @@ void SkylineRectanglePacker::addSkylineLevel(int skylineIndex,
   newSegment.x_ = x;
   newSegment.y_ = y + height;
   newSegment.width_ = width;
-  skyline_.insert(std::next(skyline_.begin(), skylineIndex), newSegment);
+  skyline_.insert(skyline_.begin() + skylineIndex, newSegment);
 
   FML_DCHECK(newSegment.x_ + newSegment.width_ <= this->width());
   FML_DCHECK(newSegment.y_ <= this->height());
 
   // delete width of the new skyline segment from following ones
-  for (int i = skylineIndex + 1; i < (int)skyline_.size(); ++i) {
-    // The new segment subsumes all or part of skyline_[i]
+  for (auto i = skylineIndex + 1; i < skyline_.size(); ++i) {
+    // The new segment subsumes all or part of fSkyline[i]
```

Comment:
If fSkyline referring to the right ivar?

Reply:
Fixed